teuthology/salt.py

```diff
@@ -99,23 +106,26 @@ def __preseed_minions(self):
         for rem in self.remotes.iterkeys():
             minion_id = rem.hostname
 
-            self.master_remote.run(args = ['sudo', 'cp',
+            self.master_remote.run(args=[
+                'sudo', 'cp',
                 'salt/minion-keys/{}.pub'.format(minion_id),
                 '/etc/salt/pki/master/minions/{}'.format(minion_id)])
-            self.master_remote.run(args = ['sudo', 'chown', 'ubuntu',
-                "salt/minion-keys/{}.pem".format(minion_id),
-                "salt/minion-keys/{}.pub".format(minion_id)])
+            self.master_remote.run(args=[
+                'sudo', 'chown', 'ubuntu',
+                'salt/minion-keys/{}.pem'.format(minion_id),
+                'salt/minion-keys/{}.pub'.format(minion_id)])
             # copy the keys via the teuthology VM. The worker VMs can't ssh to
             # each other. scp -3 does a 3-point copy through the teuhology VM.
-            sh('scp -3 {}:salt/minion-keys/{}.* {}:'.format(self.master_remote.name,
+            sh('scp -3 {}:salt/minion-keys/{}.* {}:'.format(
+                self.master_remote.name,
                 minion_id, rem.name))
```

Comment:
I actually think witht his particular line, this style is better. Also left out the trailing comma, since this list is not expected to be extended anytime soon, neither would it be as simple as adding a line (a placeholder in the string would have to be added too).